examples/exports/index.rsh

```diff
@@ -0,0 +1,20 @@
+'reach 0.1';
+
+export const a = [1, 2, [3, 4, [5, 6, 7]]];
+
+export const o = {
+  a: 5,
+  b: {
+    c: true,
+    d: [1, 2, [true, false]]
+  }
+}
+
+export const add1_impl = (x) => x + 1;
+
+export const add1= is(add1_impl, Fun([UInt], UInt));
```

Comment:
Can you add an example that use `Refine` to impose constraints on both the input and the output and that uses `assert` inside?

Reply:
added an example function that uses refine and assert `subMul2`